css/css-scroll-snap/scroll-target-padding-002.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<meta charset="utf-8">
+<title>focus() and scroll-snap-padding with snapping off (y-axis)</title>
```

Comment:
The title says "focus()" but the test uses "scrollIntoView()"

Reply:
Fixed.